.gitignore

```diff
@@ -1,2 +1,3 @@
 .DS_Store
 bin/
+docs/html
```

Comment:
IMO this should either be rooted `/docs/html/` to ignore a specific directory, or `html/` to ignore any directory named "html".  "any file named 'html' in any directory named 'docs'" feels like a weird bad mixup of specific and vague to me.

Reply:
Switched to rooted.